pkg/operator/controller.go

```diff
@@ -121,6 +128,40 @@ func (c *Controller) createOrUpdateResources(cr *imageregistryv1.Config) error {
 	return nil
 }
 
+// localCABundleHash returns the hash for the current CA bundle file.
+//
+// Opens CA bundle file and calculate the SHA256 of it, returning it as a
+// string.
+func (c *Controller) localCABundleHash() (string, error) {
+	bundleContent, err := ioutil.ReadFile(
+		fmt.Sprintf("%s/%s", caBundlePath, caBundleFileName),
+	)
```

Comment:
You should `os.Stat` the file first - at the moment the ca-bundle will not be a required file, and could be missing. If missing return an empty hash.